tests/setup.py

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+
+from setuptools import setup
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setup(
+    name="ccftools",
+    version="0.11.9",
```

Comment:
The version of the PyPi package is an interesting one. Ideally, we want it in sync with the CCF release (and we can do so by reading from the metadata of the repo/directory, just like for cmake).

However, it would mean that we cannot release an update of this package _without_ having to release a new version of CCF.

Reply:
I think they should stay on the same release cycle at first, doing a release where just the tools changed doesn't sound too bad.